# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/ktcpvs/username_make_install.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./userspace/Makefile.orig	Mon Feb 24 15:31:42 2003
+++ ./userspace/Makefile	Mon Feb 24 19:04:40 2003
@@ -14,7 +14,8 @@
 CC	= gcc
 CFLAGS	= -Wall -Wunused -Wstrict-prototypes -g -O2
 SBIN    = $(BUILD_ROOT)/sbin
-MAN     = $(BUILD_ROOT)/usr/man/man8
+MANDIR	= $(BUILD_ROOT)/usr/man
+MAN     = $(MANDIR)/man8
 MKDIR   = mkdir
 INSTALL = install
 INCLUDE = -I/usr/src/linux/include -I.. -I.
